cli/cook/querying.py

```diff
@@ -72,6 +67,30 @@ def satisfy_pred():
     return entities
 
 
+def partition(l, n):
```

Comment:
Nitpick: should this be moved to `util.py`

Reply:
Done